bin/v-add-sys-ssh-jail

```diff
@@ -29,13 +29,32 @@ if [ ! -x /sbin/jk_init ]; then
 	exit
 fi
 
+# Checking if bubblewrap is installed
+if [ ! -x /bin/bwrap ]; then
+	exit
+fi
+
 # Perform verification if read-only mode is enabled
 check_hestia_demo_mode
 
 #----------------------------------------------------------#
 #                       Action                             #
 #----------------------------------------------------------#
 
+# Move jailbash to /usr/sbin
+if [ ! -x /usr/sbin/jailbash ]; then
+	cp -f $HESTIA_COMMON_DIR/bubblewrap/jailbash /usr/sbin/jailbash
+	cp -f $HESTIA_COMMON_DIR/bubblewrap/bwrap-userns-restrict /etc/apparmor.d/bwrap-userns-restrict
```

Comment:
I assume this is only an issue with Ubuntu 24.04?  in that case we should only do it for that version

Reply:
Well I figured it wouldn't be so bad to have the same apparmor config for both Ubuntu 22.04 and Debian. If I understand correctly it will restrict some privileges there if the config is supplied.